packages/examples/markdown/src/template.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CUSTOM TEMPLATE: {{title}}</title>
```

Comment:
Nit: the hightlight.js stylesheet needs to be loaded

Reply:
Do we still support loading css from node_modules in html? Couldn't get it to work

Reply:
Well `~/node_modules/..` should work. But that wouldn't work anymore with PnP

Reply:
That's unfortunate we should probably support node modules and update the example after that. Maybe like css `~node-module-name/...`